exporter/datadogexporter/stats.go

```diff
@@ -0,0 +1,79 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !windows
+
+package datadogexporter
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/trace/pb"
+	"github.com/DataDog/datadog-agent/pkg/trace/stats"
+)
+
+const (
+	statsBucketDuration int64 = 1e10 // 10 seconds
```

Comment:
keeping it explicitly as int64 was just easier since it has to match `pushTime` here https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/1266/files#diff-9dabb193030fe7865f54b404f3e72c4e8d3289b1e5ab3c8fc6c076e2253416cbR33 which is the same type  (it's the unix nano timestamp).

Reply:
Alright, but you could still have something like `int64(10 * time.Second)`, so that you can get rid of the comment while still easily telling that this is 10s.